007-disable-usage-stats.md

```diff
@@ -0,0 +1,118 @@
+# Ability to Disable Usage Stats <!-- What do you want to call your `awesome_feature`? -->
+
+- Implementation Owner: @lohanidamodar
+- Start Date: 12-01-2021
+- Target Date: Unknown
+- Appwrite Issue: https://github.com/appwrite/appwrite/issues/547
+
+## Summary
+
+[summary]: #summary
+
+<!-- Brief explanation of the proposed contribution. Write your answer below. -->
+Due to difficulty in deployment of influxdb and telegraf in services like Heroku, in the Appwrite lite version, usage stats has to be disabled. So, want to introduce a environment variable that will allow disabling usage stats completely there by influxdb and telegraf dependency will no longer be required.
+
+## Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+**What problem are you trying to solve?**
+
+<!-- Write your answer below. -->
+
+Making Appwrite lite version easily deployable to 1 click deployment platforms like Heroku, where there is no easy way to support influxdb and telegraf that are used to record and display usage stats. So, allowing to disable this and disabling this in the Appwrite lite version will enable easy deployment to platforms like Heroku.
+
+**What is the context or background in which this problem exists?**
+
+<!-- Write your answer below. -->
+
+Supporting one click deployments to free tiers of platforms like Heroku, will help a lot in Appwrite adoption. This will allow any beginners to easily instiantiate an Appwrite paltform and see it in action, even use it in production in an envorinment with less resources.
+
+**Once the proposal is implemented, how will the system change?**
+
+<!-- Write your answer below. -->
+
+Users will be able to disable the usage stats feature completely. This will disable the display of usage statistics graph in dashboard. But this will not change any other parts of Appwrite or it's API.
+
+<!-- Please avoid discussing your proposed solution. -->
+
+## Design proposal (Step 2)
+
+[design-proposal]: #design-proposal
+
+### Introduce new environment variable
+Introduce a new environment variable to allow disabling usage stats.
+    - _APP_USAGE_STATS = enabled | disabled
```

Comment:
The first name might make most sense to me. If we'll have more options in the future we can just use new env var names like: `_APP_USAGE_STATS_OPTIONX` or `_APP_USAGE_STATS_OPTIONY`

Reply:
Updated.